### PR TITLE
Restores outputs to expected format

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: tests/create_s3_keystore/main.tf [hunk 1]
```diff
@@ -26,3 +26,7 @@ module "create_s3_keystore" {
   }
   bucket_name = aws_s3_bucket.this.id
 }
+
+output keystore {
```
Comment (easttimor) [inline tests/create_s3_keystore/main.tf:30]: description

File: tests/create_ssm_keystore/main.tf [hunk 2]
```diff
@@ -27,3 +27,7 @@ module "create_s3_keystore" {
   kms_key_id  = aws_kms_key.this.arn
   bucket_name = random_id.this.hex
 }
+
+output keystore {
```
Comment (easttimor) [inline tests/create_ssm_keystore/main.tf:31]: description

**Review by easttimor — CHANGES_REQUESTED** [review]

File: tests/create_s3_keystore/main.tf (hunk 1 above)
Comment (lorengordon) [reply tests/create_s3_keystore/main.tf:30]: we don't bother with output descriptions for test configs... the outputs are just there to let us review the outputs of the tested module(s).

**Comment by lorengordon** [comment]
Results of `make test`:
```
PASS
ok      terraform-aws-tardigrade-keystore/tests 74.229s
[terratest/test]: Completed successfully!
```

**Review by easttimor — APPROVED** [review]

**Pull request merged**